components/Download.vue

```diff
@@ -53,13 +53,11 @@ export default class extends Vue {
   @Prop({ type: String, default: "" })
   public readonly platform: string;
 
-  @Prop({ type: String, default: "" })
-  @Prop({ type: Object })
-  public readonly androidOptions: publish.AndroidApkOptions | null;
```

Comment:
Prevents the browser warning about Vue being unable to validate due to unexpected type.